src/runner/provider/qti.js

```diff
@@ -267,7 +267,12 @@ var qtiItemRuntimeProvider = {
                 done(renderingQueue);
             }, this.getLoadedClasses());
         });
-    }
+    },
+
+    getApipData: function() {
+        var apipData = this._item && _.isFunction(this._item.getApipAccessibility) && this._item.getApipAccessibility() || null;
```

Comment:
Instead of extending the default API of the item runner, you should rather dynamically add the method from the initialization, directly from the provider.  Like here: https://github.com/oat-sa/tao-test-runner-qti-fe/blob/master/src/provider/qti.js#L177